tests/phpunit/Logger/ConsoleLoggerTest.php

```diff
@@ -210,22 +232,95 @@ static function (): void {},
 
     public static function outputMappingProvider(): iterable
     {
+        yield [LogLevel::EMERGENCY, OutputInterface::VERBOSITY_QUIET, false];
+
         yield [LogLevel::EMERGENCY, OutputInterface::VERBOSITY_NORMAL, true];
 
+        yield [LogLevel::ALERT, OutputInterface::VERBOSITY_QUIET, false];
+
+        yield [LogLevel::ALERT, OutputInterface::VERBOSITY_NORMAL, true];
+
+        yield [LogLevel::CRITICAL, OutputInterface::VERBOSITY_QUIET, false];
+
+        yield [LogLevel::CRITICAL, OutputInterface::VERBOSITY_NORMAL, true];
+
+        yield [LogLevel::ERROR, OutputInterface::VERBOSITY_QUIET, false];
+
+        yield [LogLevel::ERROR, OutputInterface::VERBOSITY_NORMAL, true];
+
+        yield [LogLevel::WARNING, OutputInterface::VERBOSITY_QUIET, false];
+
         yield [LogLevel::WARNING, OutputInterface::VERBOSITY_NORMAL, true];
 
+        yield [LogLevel::NOTICE, OutputInterface::VERBOSITY_QUIET, false];
+
+        yield [LogLevel::NOTICE, OutputInterface::VERBOSITY_NORMAL, true];
+
+        yield [LogLevel::INFO, OutputInterface::VERBOSITY_QUIET, false];
+
         yield [LogLevel::INFO, OutputInterface::VERBOSITY_NORMAL, false];
 
+        yield [LogLevel::INFO, OutputInterface::VERBOSITY_VERBOSE, true];
+
+        yield [LogLevel::DEBUG, OutputInterface::VERBOSITY_QUIET, false];
+
         yield [LogLevel::DEBUG, OutputInterface::VERBOSITY_NORMAL, false];
 
-        yield [LogLevel::INFO, OutputInterface::VERBOSITY_VERBOSE, true];
+        yield [LogLevel::DEBUG, OutputInterface::VERBOSITY_VERBOSE, false];
 
         yield [LogLevel::DEBUG, OutputInterface::VERBOSITY_VERY_VERBOSE, false];
 
         yield [LogLevel::DEBUG, OutputInterface::VERBOSITY_DEBUG, true];
+    }
 
-        yield [LogLevel::ALERT, OutputInterface::VERBOSITY_QUIET, false];
+    public static function logLevelProvider(): iterable
+    {
+        yield [
+            LogLevel::EMERGENCY,
+            'message',
+            '[emergency] message',
+        ];
 
-        yield [LogLevel::EMERGENCY, OutputInterface::VERBOSITY_QUIET, false];
+        yield [
+            LogLevel::ALERT,
+            'message',
+            '[alert] message',
+        ];
+
+        yield [
+            LogLevel::CRITICAL,
+            'message',
+            '[critical] message',
+        ];
+
+        yield [
+            LogLevel::ERROR,
+            'message',
+            '[error] message',
+        ];
+
+        yield [
+            LogLevel::WARNING,
+            'message',
+            '[warning] message',
+        ];
+
+        yield [
+            LogLevel::NOTICE,
+            'message',
+            'message',
+        ];
+
+        yield [
+            LogLevel::INFO,
+            'message',
+            '[info] message',
+        ];
+
+        yield [
+            LogLevel::DEBUG,
+            'message',
+            '[debug] message',
```

Comment:
note that this test doesn't capture the formatter tags e.g. here the message that we write is `<debug>[debug] message</debug>` where the tags are handled by the output formatter. I'm don't know however how to test that...